### PR TITLE
create new classes: parainfluenza virus type 1, 2, 4 infectious disease

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
addresses #2063

### CODE REVIEW

**Comment by callahantiff** [comment]
@nicolevasilevsky - I added synonyms and dbxrefs for all 4 human parainfluenza virus infectious diseases, as specific in issue #2109.  I tried to add you as a reviewer, but for some reason you are not showing up in that list for me. Would you please take a look at this and let me know if there are any changes you would like.

One thing I did not do was run ELK, I figured it was not needed since I was only adding annotations, but maybe that's a bad assumption. Happy to run it if required, just let me know! 😄

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -434471,6 +434474,51 @@ is_a: MONDO:0005896 ! Paramyxoviridae infectious disease
...
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2063 xsd:string
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100205
+name: parainfluenza virus type 4 infectious disease
+def: "A disease caused by infection with parainfluenza virus type 4." [https://orcid.org/0000-0002-8169-9049, MONDO:patterns/infectious_disease_by_agent, PMID:21636318, PMID:27197630]
+comment: Note - this is purposely unaxiomitized because it does not align with NCBI taxonomy.
+synonym: "HPIV-4" RELATED []
+synonym: "Infection caused by human parainfluenza virus 4" EXACT []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:434512]: see above

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -434471,6 +434474,51 @@ is_a: MONDO:0005896 ! Paramyxoviridae infectious disease
...
+synonym: "Infection caused by parainfluenza virus 4" RELATED []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:434513]: see above

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -434471,6 +434474,51 @@ is_a: MONDO:0005896 ! Paramyxoviridae infectious disease
...
+synonym: "Infection due to human parainfluenza virus 4" EXACT []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:434514]: see above

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -434471,6 +434474,51 @@ is_a: MONDO:0005896 ! Paramyxoviridae infectious disease
...
+synonym: "Infection due to parainfluenza virus 4" RELATED []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:434515]: see above

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -434471,6 +434474,51 @@ is_a: MONDO:0005896 ! Paramyxoviridae infectious disease
...
+xref: omop: 4288743 {source="MONDO:equivalentTo"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:434516]: see above

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -434471,6 +434474,51 @@ is_a: MONDO:0005896 ! Paramyxoviridae infectious disease
...
+xref: snomed:68949000 {source="MONDO:equivalentTo"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:434517]: see above

**Comment by nicolevasilevsky** [comment]
>  I tried to add you as a reviewer, but for some reason you are not showing up in that list for me. 

I think that's because I created this PR, I guess I'm not allowed to review my own PR :)

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -422271,6 +422271,7 @@ id: MONDO:0042974
 name: parainfluenza virus type 3 infectious disease
 def: "Parainfluenza virus type 3 is one of a group of common viruses known as human parainfluenza viruses (HPIV) that cause a variety of respiratory illnesses. Symptoms usually develop between 2 and 7 days from the time of exposure and typically resolve in 7-10 days. Symptoms may include fever, runny nose, and cough. HPIV-3 can also cause bronchiolitis, bronchitis, and pneumonia. Infants and young children are particularly susceptible to HPIV-3 infections, though older adults and those with a weakened immune system are also at risk for complications. HPIVs are usually spread from an infected person to others through coughing, sneezing, and/or touching. There is currently no vaccine to protect against parainfluenza virus infections. Most HPIV infections resolve on their own and do not require special treatment, though medical intervention may be necessary for severe breathing problems. Most adults have antibodies against parainfluenza but can get repeat infections." [GARD:0004215]
 subset: gard_rare
+synonym: "HPIV-3" RELATED []
```
[the remark this answers is not in the document]
Comment (callahantiff) [reply src/ontology/mondo-edit.obo:422274]: Thanks so much for the clarification. I'm sorry about all of the errors. I removed `HPIV-3` as it is not from OMOP.

[the remark this answers is not in the document]
Comment (callahantiff) [reply src/ontology/mondo-edit.obo:422274]: I noticed some of the existing synonyms were not lowercased, I went ahead and made sure they all were (except abbreviations) to be consistent. Hope that's OK :)

[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422274]: Don't apologize - it's a steep learning curve! This is how I learned too - I made so many mistakes!

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -422282,7 +422283,9 @@ synonym: "infection due to parainfluenza virus 3" RELATED []
 synonym: "PIV3" RELATED [GARD:0004215]
 xref: GARD:0004215 {source="MONDO:equivalentTo"}
 xref: ICD9:079.89 {source="linkedlifedata"}
+xref: omop: 4147524 {source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (callahantiff) [reply src/ontology/mondo-edit.obo:422286]: That makes sense!

**Review by callahantiff — COMMENTED** [review]
Thanks for your very helpful feedback 😄 . I made all requested changes. Will make commit now.

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -422271,17 +422271,18 @@ id: MONDO:0042974
...
-synonym: "Human respirovirus 3 disease or disorder" EXACT []
-synonym: "Human respirovirus 3 infectious disease" EXACT []
-synonym: "infection caused by human parainfluenza virus 3" RELATED []
-synonym: "infection caused by parainfluenza virus 3" RELATED []
-synonym: "infection due to human parainfluenza virus 3" EXACT []
-synonym: "infection due to parainfluenza virus 3" RELATED []
+synonym: "human parainfluenza virus type 3" RELATED [GARD:0004215]
+synonym: "human respirovirus 3 caused disease or disorder" EXACT [MONDO:patterns/specific_infectious_disease_by_agent]
+synonym: "human respirovirus 3 disease or disorder" EXACT []
+synonym: "human respirovirus 3 infectious disease" EXACT []
```
Comment (callahantiff) [inline src/ontology/mondo-edit.obo:422277]: Synonyms on lines 422275-422277 were not added by me, I just made them lower case/

**Comment by cmungall** [comment]
thanks all!

Looks like we don't have links to ncbitaxon for the child nodes? Did we request these from ncbi also?

**Comment by nicolevasilevsky** [comment]
> Looks like we don't have links to ncbitaxon for the child nodes? Did we request these from ncbi also?

I did, I haven't heard back from them yet.

File: src/ontology/mondo-edit.obo (hunk 9 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422277]: thanks!

**Comment by nicolevasilevsky** [comment]
thanks for all your changes, Tiffany, this all looks good to me.

@cmungall I will go ahead and merge this unless you have any objections.

**Comment by nicolevasilevsky** [comment]
@callahantiff 

As NCBI taxonomy said:
As to the name 'human parainfluenza virus, the problem is that no virus by that name exists. Instead it is a loose reference to four distinct viruses in the family Paramyxoviridae that cause respiratory infections particular in younger children.
The Taxonomy staff have tagged all four as 'human parainfluenza virus' and a query in the taxonomy browser will now offer you to choose among the four the one you are really working with:
human parainfluenza virus
Human respirovirus 1 [in-part: human parainfluenza virus]
Human respirovirus 3 [in-part: human parainfluenza virus]
Human rubulavirus 2 [in-part: human parainfluenza virus]
Human rubulavirus 4 [in-part: human parainfluenza virus] 

Would you be okay if we deleted the new class 'parainfluenza infectious disease' ?

**Comment by cmungall** [comment]
Let's just make a union definition

**Comment by nicolevasilevsky** [comment]
> Let's just make a union definition

@cmungall sounds like a good idea, but I realized I don't know how to do that. If I add an equivalence axiom that is all the subtypes, it causes a reasoning error:

equiv axiom:
NCBITaxon_12730 and NCBITaxon_1979160 and 'Human respirovirus 3' and NCBITaxon_1979161

image

**Comment by callahantiff** [comment]
> > Let's just make a union definition
> 
> @cmungall sounds like a good idea, but I realized I don't know how to do that. If I add an equivalence axiom that is all the subtypes, it causes a reasoning error:
> 
> equiv axiom:
> NCBITaxon_12730 and NCBITaxon_1979160 and 'Human respirovirus 3' and NCBITaxon_1979161
> 
> image

Please ignore me if this is distracting, but I am also trying to make sure I understand 😄 . @cmungall, would the class for `human parainfluenza virus` be created by using `owl:UnionOf` with each of the concepts @nicolevasilevsky describes?

**Comment by nicolevasilevsky** [comment]
@callahantiff yeah, that's what I understand, but it didn't work when I tried to do it above, unless I did it wrong.

**Comment by matentzn** [comment]
What chris means by a union is a list of terms with OR in between

```
A OR B OR C
```

However, always think patterns. This time you want to look at:
https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/patterns/dosdp-patterns/infectious_disease_by_agent.yaml

So the correct EQ would be:

MONDO:001 = "'infectious disease' and 'realized in response to stimulus' some (NCBITaxon_12730 or NCBITaxon_1979160 or 'Human respirovirus 3' or NCBITaxon_1979161)"

However, we _do not want to do that_ if it can be avoided. So what he (I assume) really wants is this:

```
parainfluenza infectious disease = "parainfluenza infectious disease type 1" or "parainfluenza infectious disease type 2" or "parainfluenza infectious disease type 3" or "parainfluenza infectious disease type 4"
```

**Comment by matentzn** [comment]
BTW: an intersection is a statement of the sort `A AND B AND C` (in contrast to `A or B or C` which is a union)

**Comment by callahantiff** [comment]
> @callahantiff yeah, that's what I understand, but it didn't work when I tried to do it above, unless I did it wrong.

Awesome! I always do this type of stuff programmatically, so I'm not much a help when it comes to adding things via Protége.

**Comment by nicolevasilevsky** [comment]
@matentzn this still doesn't work, am I doing it wrong?

image

**Comment by cmungall** [comment]
the parent disease is the union of the child diseases, not child taxa

**Comment by nicolevasilevsky** [comment]
Oooohh!! Thanks!

**Comment by nicolevasilevsky** [comment]
@callahantiff I just unassigned you from this because everything is done here. I am going to merge once the test passes.

**Comment by callahantiff** [comment]
> @callahantiff I just unassigned you from this because everything is done here. I am going to merge once the test passes.

Great. Thanks Nicole and sorry this was so complicated and I was not very helpful. I hope to be more helpful moving forward (if you'll have me 😄 ). You are a great teacher!

**Comment by nicolevasilevsky** [comment]
This was great, @callahantiff, I learned a lot from this! The complicated tickets are the best ones to learn from.

**Comment by cmungall** [comment]
yes, it really helps us build up patterns docs

**Pull request merged**